packages/story-editor/src/components/floatingMenu/elements/dismiss.js

```diff
@@ -19,27 +19,42 @@
  */
 import { Icons } from '@googleforcreators/design-system';
 import { __ } from '@googleforcreators/i18n';
+import { trackEvent } from '@googleforcreators/tracking';
+import PropTypes from 'prop-types';
 
 /**
  * Internal dependencies
  */
 import { useFloatingMenu } from '../context';
-import { IconButton } from './shared';
+import { IconButton, useProperties } from './shared';
 
-function Dismiss() {
+function Dismiss({ isMultiple }) {
+  const { type } = useProperties(['type']);
   const { handleDismiss } = useFloatingMenu(
     ({ actions: { handleDismiss } }) => ({
       handleDismiss,
     })
   );
 
+  const handleClick = () => {
+    trackEvent('floating_menu', {
+      name: 'dismiss_menu',
+      element: isMultiple ? 'multiple' : type,
+    });
+    handleDismiss();
```

Comment:
💭  May be good to pass the click event here in case it is needed in the future
```suggestion
  const handleClick = (evt) => {
    trackEvent('floating_menu', {
      name: 'dismiss_menu',
      element: isMultiple ? 'multiple' : type,
    });
    handleDismiss(evt);
```

Reply:
let's add it if we need it later? Right now it's unnecessary.